tests/testthat/test-client.R

```diff
@@ -1,17 +1,15 @@
 test_that("Default connection works", {
-    wfs <- emodnet_init_wfs_client(service = "seabed_habitats_individual_habitat_map_and_model_datasets")
-  expect_equal(class(wfs),
-               c("WFSClient", "OWSClient", "OGCAbstractObject", "R6"))
-  expect_equal(wfs$getUrl(),
-               "https://ows.emodnet-seabedhabitats.eu/geoserver/emodnet_open_maplibrary/wfs")
- })
+    wfs <- create_biology_wfs()
+    expect_equal(class(wfs), c("WFSClient", "OWSClient", "OGCAbstractObject", "R6"))
+    expect_equal(wfs$getUrl(), "http://geo.vliz.be/geoserver/Emodnetbio/wfs")
+})
 
 test_that("Specified connection works", {
```

Comment:
@annakrystalli what was the difference between default and specified connection? I'm not sure I got it so I might have badly altered the test.